src/aura/BGE_ConfigurationWizard/BGE_ConfigurationWizardHelper.js

```diff
@@ -887,6 +892,15 @@
                 this.onMetadataUpdated.notify();
             }
 
+            /**
+             * @description sets the pendingsave flag to disable Save button so duplicates can't be created
+             * @return void.
+             */
+            function togglePendingSave() {
+                this.pendingSave = !this.pendingSave;
```

Comment:
Would it be safer to explicitly set pendingSave to true or false? Is there a condition like clicking twice that could cause the state to toggle back to false incorrectly?